share/picongpu/examples/WeibelTransverse/include/picongpu/param/png.param

```diff
@@ -37,7 +37,7 @@ namespace picongpu
     {
         // normalize EM fields to typical laser or plasma quantities
         //-1: Auto:     enable adaptive scaling for each output
-        // 1: Laser:    typical fields calculated out of the laser amplitude
+        // 1: Laser:    [outdated]
         // 2: Drift:    [outdated]
         // 3: PlWave:   typical fields calculated out of the plasma freq.,
         //              assuming the wave moves approx. with c
```

Comment:
why is blowout below not set to `[outdated]`?

Reply:
It is missing here and therefore not found by search and replace

Reply:
will do

Reply:
done